tldraw/apps/tldraw-logseq/package.json

```diff
@@ -32,7 +32,9 @@
     "tsup": "^6.1.2",
     "typescript": "^4.7.3",
     "zx": "^6.2.4",
-    "@radix-ui/react-switch": "^0.1.6-rc.40"
+    "@radix-ui/react-switch": "^0.1.6-rc.40",
```

Comment:
It seems you haven't checked in the `yarn.lock` file after installing new dependencies